### PR TITLE
fix(tests): Fixed chat tests

### PR DESCRIPTION
**Pull request by igor-sirotin**
Fixes https://github.com/status-im/status-desktop/issues/7112
Requires https://github.com/status-im/StatusQ/pull/869

### What does the PR do

This fixes chat tests, which partially got broken after merging https://github.com/status-im/status-desktop/pull/6508

### Affected areas

tests

### CODE REVIEW

File: test/ui-test/src/screens/StatusChatScreen.py [hunk 1]
```diff
@@ -57,16 +62,24 @@ def send_message(self, message: str):
...
     # Verifications region:        
     def verify_last_message_sent(self, message: str):
-        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
-        verify(loaded, "Checking last message sent: " + message)
-        verify_text_contains(str(last_message_obj.text), str(message))
+        # Get the message text
+        # We don't search for StatusTextMessage_chatText directly, because there're 2 instances of it in a reply message
+        last_message_obj = self.get_message_at_index(0)
+        text_message_obj = getChildrenWithObjectName(last_message_obj, "StatusMessage_textMessage")[0]
+        text_edit_obj = getChildrenWithObjectName(text_message_obj, "StatusTextMessage_chatText")[0]
```
Comment (jrainville) [inline test/ui-test/src/screens/StatusChatScreen.py:69]: It,s better to put those names in an enum like the others at the top

**Review by jrainville — APPROVED** [review]
Good job. Just add the enum too be cool

File: test/ui-test/src/screens/StatusChatScreen.py (hunk 1 above)
Comment (jrainville) [reply test/ui-test/src/screens/StatusChatScreen.py:69]: If you prefer, you can create another `Class` for just constant names then.

Comment (noeliaSD) [reply test/ui-test/src/screens/StatusChatScreen.py:69]: I agree it is better to add them into an enum. A specific class for constants is fine!!

File: test/ui-test/src/screens/StatusChatScreen.py [hunk 2]
```diff
@@ -57,16 +62,24 @@ def send_message(self, message: str):
...
+        verify_text_contains(str(text_edit_obj.text), str(message))
 
     def verify_last_message_sent_is_not(self, message: str):
-        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
-        if not loaded:
+        chat_log_obj = wait_and_get_obj(ChatComponents.CHAT_LOG.value)
+        last_message_obj = chat_log_obj.itemAtIndex(int(0))
+        text_message_objs = getChildrenWithObjectName(last_message_obj, "StatusMessage_textMessage")
+        if len(text_message_objs) == 0:
             test.passes("Success: No message was found")
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusChatScreen.py:78]: Not related to this pr but seen now.. It should be a call to the `SquishVerificationDriver` not a direct call to squish api. Could you kindly update it?

File: test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def step(context):
         _statusChat.send_message(row[0])
         _statusChat.verify_last_message_sent(row[0])
 
-@Then("user is able to send  a random chat message")
+@Then("user is able to send a random chat message")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py:43]: We agreed on this so you could, as well, refactor this!
```suggestion
@Then("the user is able to send a random chat message")
```

**Review by noeliaSD — APPROVED** [review]
LGTM, and tested! Just minor comments added!

**Pull request merged**